QtCollider/primitives/prim_misc.cpp

```diff
@@ -167,7 +167,7 @@ QC_LANG_PRIMITIVE( Qt_SetGlobalPalette, 1, PyrSlot *r, PyrSlot *a, VMGlobals *g
 {
   if( !QcApplication::compareThread() ) return QtCollider::wrongThreadError();
 
-  QPalette p = QtCollider::get( a );
+  QPalette p = (QPalette&&) QtCollider::get(a);
```

Comment:
This is the error produced by VS with the original:
  C:\Users\Rainer\Projects\sc\supercollider\QtCollider\primitives\prim_misc.cpp(170): error C2440: 'initializing' : can
not convert from 'QtCollider::DecodableSlot' to 'QPalette' [C:\Users\Rainer\Projects\sc\supercollider\build3.7VS64\lang
\libsclang.vcxproj]